README.md

```diff
@@ -101,10 +124,37 @@ cd /usr/local/src/php-driver
 
 ## Copyright
 
-Copyright 2015-2016 DataStax, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
-
-[http://www.apache.org/licenses/LICENSE-2.0](http://www.apache.org/licenses/LICENSE-2.0)
-
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+Copyright 2015-2017 DataStax, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+[Apache Cassandra]: http://cassandra.apache.org
+[DSE PHP driver]: http://docs.datastax.com/en/developer/php-driver-dse/latest
+[DataStax Enterprise]: http://www.datastax.com/products/datastax-enterprise
+[DataStax C/C++ Driver for Apache Cassandra]: http://docs.datastax.com/en/developer/cpp-driver/latest
+[DataStax download server]: http://downloads.datastax.com/php-driver
+[GitHub]: https://github.com/datastax/php-driver
+[Home]: http://docs.datastax.com/en/developer/php-driver/latest
+[API]: http://docs.datastax.com/en/developer/php-driver/latest/api
+[Provide your input]: http://goo.gl/forms/HbSiIJ2tLP
+[installing-details]: https://github.com/datastax/php-driver/blob/master/ext/README.md
+[contribution-policy]: https://github.com/datastax/php-driver/blob/master/CONTRIBUTING.md
+[Behat Framework]: http://docs.behat.org
+[feature-examples]: /features
+[`duration`]: http://docs.datastax.com/en/developer/php-driver/latest/api/Cassandra/class.Duration
+[simple string]: /features/simple_string_queries
```

Comment:
These feature links are broken when viewing the README on GitHub. I think if you add `.feature` to them and add a documentor rewrite rule to remove then it would work for both.